frontend/src/components/organism/Projections.tsx

```diff
@@ -79,35 +84,23 @@ const Projections: React.FC<{ className?: string }> = ({ className }) => {
         description="View the projected liquidity of your company."
       />
 
-      <div className="projection-row projection-header">
+      <ProjectionRowEntry
+        type={undefined}
+        gapAbove={false}
+        className="projection-header"
+      >
         <strong>Date</strong>
         <strong>Description</strong>
         <strong>Income</strong>
         <strong>Expense</strong>
         <strong>Balance</strong>
-      </div>
+      </ProjectionRowEntry>
       <div className="projection-table">{renderTransactions()}</div>
```

Comment:
This could also be extracted to a `ProjectionTable` `styled.div` to increase reusability and atomicity (I hope that's a word ;). Not necessary though.